README.md

```diff
@@ -88,8 +88,9 @@ nqmindsTDX.authenticate("myTokenID","myTokenSecret", function(err, accessToken)
     });
 
     // Delete dataset data
-    // E.g. primaryKey="timestamp" keyValue="1093290930"
-    nqmindsTDX.deleteDatasetData("datasetId, "primaryKey", keyValue, function(err, response)) {
+    // data should take the form of an array e.g.
```

Comment:
data needn't be an array - it will be converted to an array if it isn't supplied as one - to support single value deletes